tools/kvrocks2redis/parser.cc

```diff
@@ -54,7 +54,7 @@ Status Parser::parseSimpleKV(const Slice &ns_key, const Slice &value, int expire
 
 Status Parser::parseComplexKV(const Slice &ns_key, const Metadata &metadata) {
   RedisType type = metadata.Type();
-  if (type < kRedisHash || type > kRedisBitmap) {
+  if (type < kRedisHash || type > kRedisSortedint) {
```

Comment:
Since `Sortedint` is a new addition to Kvrocks and is not supported by Redis, we don't need to deal with it here.

Reply:
@caipengbo Seems we have converted the sorted int to the Redis sort type, so it should make sense here?

Reply:
a configuration should be add to support such compatiable convert?

Reply:
If it is possible, we should support to convert the sorted int to the Redis set type, we shouldn't let user to handle this problem.

Reply:
> If it is possible
this also means user could read sorted int by redis commands after migrating from kvrocks to redis

Reply:
yeah, @karelrooted should have converted to the Redis set if my memory serves. Can u help to confirm? @caipengbo

Reply:
I didn't find it converted to Redis set.